### DISCUSSION TITLE
mitmproxy not in path after homebrew install on MacOS

### DISCUSSION THREAD

**Discussion by sageacy** [mitmproxy/mitmproxy#5915] [Q&A] [answered]
I installed mitmproxy with the recommended homebrew installation in zsh on MacOS

`brew install mitmproxy`

But after the successful installation, my path does seem to have been configured with mitmproxy, and I don't know where it was installed to run it directly.

**Comment by sageacy** [comment, marked as answer]
Operator error... I installed it under a different shell from where I was trying to run it.

I also found where the binaries are installed on MacOS:  /opt/homebrew/bin

**Reply by mhils** [reply]
Thanks for posting a follow-up! 😃
